docs/fr_FR/support.md

```diff
@@ -11,8 +11,11 @@ Une petite aide, qui devrait vous aider à créer une demande d'assistance.
 
 ::: tip
 Si vous avez besoin d'aide, décrivez votre problème aussi précisément que possible. Plus vous décrirez votre problème de manière précise et détaillée, plus nous pourrons vous aider rapidement et efficacement.:::
+:::
+
 ::: warning
```

Comment:
remove the `:::` at the end of line :wink:

Reply:
ok thank